src/test/java/org/vanilladb/core/storage/buffer/BufferPoolConcurrencyTest.java

```diff
@@ -52,17 +53,49 @@ public static void finish() {
 			logger.info("FINISH BUFFER POOL CONCURRENCY TEST");
 	}
 
+	@Test
+	public void testSwapping() {
+		int clinetCount = BUFFER_COUNT * 2;

```

Comment:
1.  There seems a typo of "client".
2.  Does it mean 1-to-1 mapping of the clients and the buffers.